src/components/Navbar/SideWidgets/CurrentSpeed.vue

```diff
@@ -1,12 +1,40 @@
 <script setup lang="ts">
 import SpeedCard from '@/components/Core/SpeedCard.vue'
+import SpeedCardClickable from '@/components/Core/SpeedCardClickable.vue'
 import { useMaindataStore, useVueTorrentStore } from '@/stores'
 import { storeToRefs } from 'pinia'
 import { useI18nUtils } from '@/composables'
+import { useTorrentStore } from '@/stores'
+import { TorrentState } from '@/constants/vuetorrent'
+import { computed } from 'vue'
```

Comment:
Sort and merge imports

```suggestion
import SpeedCard from '@/components/Core/SpeedCard.vue'
import SpeedCardClickable from '@/components/Core/SpeedCardClickable.vue'
import { useI18nUtils } from '@/composables'
import { TorrentState } from '@/constants/vuetorrent'
import { useMaindataStore, useTorrentStore, useVueTorrentStore } from '@/stores'
import { storeToRefs } from 'pinia'
import { computed } from 'vue'
```